src/active_active_tests/orchestrator.py

```
@@ -0,0 +1,324 @@
+SRC_DIR = os.path.dirname(TEST_DIR)
+VALKEY_SERVER_PATH = os.path.join(SRC_DIR, "valkey-server")
+VALKEY_CLI_PATH = os.path.join(SRC_DIR, "valkey-cli")
+PROXY_PATH = os.path.join(TEST_DIR, "proxy.py")
+
+REPORT_PATH = os.environ.get("VALKEY_TEST_REPORT_PATH", os.path.join(TEST_DIR, "active_active_test_report.md"))
+
+def cleanup_processes():
+    print("Cleaning up old valkey and proxy processes...")
+    for port in [7000, 7001, 8000, 8001]:
+        try:
+            output = subprocess.check_output(["lsof", "-t", f"-i:{port}"]).decode().strip()
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)